src/NodeServices/IAnalyticsClient.cs

```diff
@@ -433,7 +433,27 @@ public enum Actions
         /// <summary>
         /// ExportSettings event, e.g. tracks the ExportSettings event
         /// </summary>
-        ExportSettings
+        ExportSettings,
```

Comment:
Now that I saw this, I think the last two actions we added were misleading. Would you also update these to be `Import` and `Export` only (removing the `Settings`). The action is supposed to be generic so that they can be reused in other categories

Reply:
The ExportSettings option and ImportSettings option were renamed to Export and Import (so the method calls were also updated). See the next commit:
https://github.com/DynamoDS/Dynamo/pull/13581/commits/0655f6d242fa449dc828a75d8634567d3b5737f4